1-mandatory/2-people-i-know.js

```diff
@@ -402,6 +427,16 @@ This time, I only want the full names of the people who can multitask
 */
 
 let colleaguesWhoCanMultitask = [];
+for (let i = 0; i < friends.length; i++) {
+  const friend = friends[i];
+  for (let j = 0; j < friend.colleagues.length; j++) {
+    const colleague = friend.colleagues[j];
+    if (colleague.skills.includes("Multi-tasking")) {
+      colleaguesWhoCanMultitask.push(`${colleague.name}`);
+    }
+  }
+}
+console.log(colleaguesWhoCanMultitask);
```

Comment:
Great and very precise solution, but I wrote it as below.
let colleaguesWhoCanMultitask = [];
for (let friend of friends){
  for (let colleague of friend.colleagues){
    if(colleague.skills.includes("Multi-tasking")){
      colleaguesWhoCanMultitask.push(colleague.name);
    }
  }
}